client/lib/jetpack/backup-utils.js

```diff
@@ -129,6 +129,25 @@ export const getDailyBackupDeltas = ( logs, date ) => {
 	};
 };
 
+export const getRawDailyBackupDeltas = ( logs, date ) => {
+	return getEventsInDailyBackup( logs, date ).filter( ( { activityName } ) =>
+		[
+			'attachment__uploaded',
+			// 'attachment__updated',
```

Comment:
Commenting because this activity type is visually similar to `attachment__uploaded` in the UI, which is misleading.